content/blog/entries/cc-vocabulary-new-design-system/contents.lr

```diff
@@ -0,0 +1,306 @@
+title: CC Vocabulary is a new design system for Creative Commons
+---
+categories:
+announcements
+cc-vocabulary
+product
```

Comment:
Could you add the `gsoc` and `gsoc-2019` categories as well?

Reply:
Done!